app/components/search/SearchResults/SearchResults.module.scss

```diff
@@ -1,85 +1,136 @@
-@import '../../../styles/utils/_helpers.scss';
+@import '~styles/utils/_helpers.scss';
 
-.searchResultsContainer {
+.searchMapContainer {
   position: relative;
+  display: flex;
+  flex-direction: row;
+  @media screen and (max-width: $break-tablet-s) {
+    flex-flow: column-reverse;
+  }
+}
+
+.searchResultsContainer {
   display: flex;
   flex-direction: column;
-  width: 100%;
+  flex-grow: 1;
+  position: relative;
+  width: 60%;
   height: 100%;
-  overflow-y: scroll;
+  margin-right: 15px;
+  @media screen and (max-width: $break-tablet-s) {
+    width: 100%;
+    z-index: 1;
+    top: 0;
+    border-top-left-radius: 12px;
+    border-top-right-radius: 12px;
+    box-shadow: 0 0 20px rgb(0 0 0 / 55%);
+    background: #fff;
+    transition: top 200ms ease-in-out;
+    &.expandList {
+      top: calc(-50vh + 20px);
+    }
+  }
+}
+
+.searchResultsTopShadow {
+  @media screen and (max-width: $break-tablet-s) {
+    box-shadow: 0 0 20px rgb(0 0 0 / 55%);
+  }
 }
 
 .searchResult {
   display: flex;
   flex-direction: row;
   align-items: stretch;
+  justify-content: space-between;
   padding-top: 21px;
   padding-bottom: 24px;
   padding-left: 24px;
-  border-left: 1px solid #F4F4F4;
-  border-bottom: 1px solid #F4F4F4;
+  border-left: 1px solid $color-grey3;
+  border-bottom: 1px solid $color-grey3;
 
-  @media screen and (max-width: 1120px) {
+  @media screen and (max-width: $break-desktop) {
     flex-direction: column;
   }
 
+  @media screen and (max-width: $break-tablet-l) {
+    padding-left: 30px;
+  }
+
+
   @media screen and (max-width: 600px) {
     padding: 18px 20px;
   }
+
+  h3 {
+    font-weight: 700;
+    font-size: 18px;
+    padding-bottom: 24px;
+  }
+
+  &_address {
+    border-bottom: 1px solid $color-grey3;
+    padding: 24px 0;
+  }
+
+  &_addressLast {
```

Comment:
I know that this isn't complete yet and that you're working on it for the next PR, but could this be replaced with [`:last-child()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:last-child)?

Reply:
I've added a to-do for now and will address in the next stage.